pinot-spi/src/main/java/org/apache/pinot/spi/config/table/IndexingConfig.java

```diff
@@ -31,6 +31,7 @@ public class IndexingConfig extends BaseJsonConfig {
    * This should be equal to the one specified in RangeIndexType.
    */
   private static final int DEFAULT_RANGE_INDEX_VERSION = 2;
+  public static final double DEFAULT_NO_DICTIONARY_SIZE_RATIO_THRESHOLD = 0.85d;
```

Comment:
nit: can you move this line above the `DEFAULT_RANGE_INDEX_VERSION` (since this is public and all the others are private)